__tests__/react.tsx

```diff
@@ -0,0 +1,208 @@
+import * as React from "react"
```

Comment:
As a standard, we're using `.spec` in our test files.
So it should be `react.spec.tsx`.
Is that ok with you?